tests/app/data/test_schema_validation.py

```diff
@@ -83,8 +75,6 @@ def validate_json_against_schema(file, json_to_validate, schema):
             return []
         except jsonschema.exceptions.ValidationError as e:
             return ["Schema Validation Error! File [{}] does not validate against schema. Error [{}]".format(file, e)]
-        except JSONDecodeError as e:
```

Comment:
I guess I could and it might lead to a cleaner error handling, but won't an unhandled exception fail the test with a stack trace anyway?